apps/remote_control/test/lexical/remote_control/search/indexer/extractors/ecto_schema_test.exs

```diff
@@ -0,0 +1,258 @@
+defmodule Lexical.RemoteControl.Search.Indexer.Extractors.EctoSchemaTest do
+  use Lexical.Test.ExtractorCase
+
+  def index(source) do
+    do_index(source, fn entry -> entry.type == :struct end)
+  end
+
+  describe "finds the structs defined by schema" do
+    test "only finds if Ecto.Schema is used" do
+      {:ok, results, _} =
+        ~q[
+        defmodule NotEcto do
+          schema "not ecto" do
+            field :ecto, :boolean, default: false
+          end
+        end
+        ]
+        |> index()
+
+      assert results == []
+    end
+
+    test "if ecto.schema is aliased" do
+      {:ok, [struct], _doc} =
+        ~q[
+         defmodule MySchema do
+           alias Ecto.Schema , as: SCM
+           use SCM
+           schema "my_schema" do
+             field :last_name, :string
+           end
+        end
+        ]
```

Comment:
There's a lot of inconsistent indentation in the code examples in these tests. Can we clean it up?

```suggestion
        ~q[
          defmodule MySchema do
            alias Ecto.Schema , as: SCM
            use SCM
            schema "my_schema" do
              field :last_name, :string
            end
          end
        ]
```

Reply:
most of the inconsistent indentation in the expected cases is due to how extract works. Since it's pulling out a range that has a surrounding indentation, but the range pulls out the context so it starts with no indentation, however, the next line still contains the context's indentation.